dashboard/config/routes.rb

```diff
@@ -393,6 +393,10 @@ module OPS
     post 'attend/:session_code/join', controller: 'workshop_enrollment', action: 'confirm_join_session'
     get 'attend/:session_code/upgrade', controller: 'session_attendance', action: 'upgrade_account'
     post 'attend/:session_code/upgrade', controller: 'session_attendance', action: 'confirm_upgrade_account'
+
+    get 'workshop_user_admin/facilitator_courses', controller: 'workshop_user_admin', action: 'facilitator_courses_form'
+    post 'workshop_user_admin/assign_course', controller: 'workshop_user_admin', action: 'assign_course'
+    get 'workshop_user_admin/remove_course', controller: 'workshop_user_admin', action: 'remove_course'
```

Comment:
Even if that were the case, I'd say it should be a `post` like the `assign_course` route, not a `get` which I'd expect not to have side effects. Luckily Rails has us covered here, and can create buttons to any controller method easily. With [button_to]( http://www.rubydoc.info/docs/rails/3.2.8/ActionView%2FHelpers%2FUrlHelper%3Abutton_to) for instance you can create a button with whatever route and httm method you want.

Even better, you can reference the route definition by controller (which can here be left out since it's the same controller) and action, like:

```
button_to 'Remove', {action: :remove_course}, course: course.course, facilitator_id: @facilitator.id
```
I'm pretty sure in this case you don't even need the specify the method, but if that doesn't work add `method: :delete`